apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/wpcom-global-styles-view.js

```diff
@@ -67,4 +68,10 @@ document.addEventListener( 'DOMContentLoaded', () => {
 		} );
 		window.location = previewButton.href;
 	} );
+
+	resetButton?.addEventListener( 'click', ( event ) => {
+		event.preventDefault();
+		recordEvent( 'wpcom_global_styles_gating_notice_reset' );
```

Comment:
Shouldn't the event be `calypso_global_styles_gating_notice_reset_support_click`? 🤔

Reply:
I wanted to follow the same naming format we're using for the other events in the frontend: `wpcom_global_styles_gating_notice_upgrade` and `wpcom_global_styles_gating_notice_preview`. We don't use the `_click` suffix in those, and there is only one word after the `wpcom_global_styles_gating_notice_` prefix.

Reply:
Fair enough!